doc/completion-nvim.txt

```diff
@@ -346,15 +346,23 @@ g:completion_matching_strategy_list	*g:completion_matching_strategy_list*
 	default value: ['exact']
 
 g:completion_matching_ignore_case	*g:completion_matching_ignore_case*
-	
+
 	Enable ignore case matching in all matching strategy. For example
 >
 	let g:completion_matching_ignore_case = 1
 <
 	default value: 0
 
+g:completion_matching_smart_case	*g:completion_matching_smart_case*
+
+	Enable smart case matching in all matching strategy. For example
+>
+	let g:completion_matching_smart_case = 1
```

Comment:
Would it make sense to change this to default to the value of `smartcase`?

```suggestion
	let g:completion_matching_smart_case = &smartcase
```

Reply:
I guess that would be ok, but it's up to @haorenW1025 .
We do not use `&ignorecase` for the `g:completion_matching_ignore_case` option though...

Reply:
> We do not use `&ignorecase` for the `g:completion_matching_ignore_case` option though...

True. But I guess that could also be a valid change to make (possibly separate from this though)?

Reply:
Hmm using `&smartcase` is a great idea, I can change `g:completion_matching_ignore_case` to use`&ignore` by default too.

Reply:
I updated it to use `&smartcase`.